object_detection.py

```diff
@@ -120,13 +120,13 @@ def show_image(file_name):
 
 def robot_say(text):
     print('Say {}'.format(text))
-    robot.say_text(text)
+    robot.behavior.say_text(text)
 
 
 def analyze():
     stand_by()
     show_camera()
-    robot_say('My lord, I found something interesting. Give me 5 seconds.')
+    robot_say('Wall-E, I found something interesting. Give me 5 seconds.')
```

Comment:
Can we not update this? Because I have to update the docs and the video :(

Reply:
Sure, I just wanted to test changing the script